.gitignore

```diff
@@ -56,4 +56,4 @@ buildbuddy-key.pem
 enterprise/config/user-configs
 
 # Bazel compact execution log
-/bazel_compact_exec_log.binpb.zst
+bazel_compact_exec_log.binpb.zst
```

Comment:
does it work to instead prefix the compact exec log file path with `%workspace%/`? (otherwise I think these files will keep building up as we run `bazel` within different dirs)

Reply:
```
ERROR: /Users/sluongng/work/buildbuddy/buildbuddy/website/%workspace%/bazel_compact_exec_log.binpb.zst (No such file or directory)
ERROR: Error initializing execution log: /Users/sluongng/work/buildbuddy/buildbuddy/website/%workspace%/bazel_compact_exec_log.binpb.zst (No such file or directory)
```

Unfortunately, the flag is a bit dumb at the moment. 
Hopefully, upstream Bazel would make it write to output base automatically instead of us having to specify the exact location.

cc: @fmeum